examples/tree/simple/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/charmbracelet/lipgloss/tree"
+)
+
+func main() {
+	t := tree.New(
+		"Root",
+		"Item 1",
+		"Item 2",
+		tree.New(
+			"Item 3",
+			"Item 3.1",
+		),
+	)
```

Comment:
```suggestion
	t := tree.New("Root").Item("Item 1", "Item 2", tree.New("Item 3").Item("3.1"))
```

Reply:
Perhaps `Item` → `Child` for trees.

Reply:
its the same api for trees and lists though... unless we embed a tree in a `type List struct` and delegate the methods to it 🤔

Reply:
done